data-service/templates/data_service.yml

```diff
@@ -0,0 +1,55 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Resources:
+
+  DynamoDBTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+     #TODO: change ddb table names to have region prefix passed in as a resource
```

Comment:
Why would we need a region prefix for the table name? We'll have separate accounts per-region, right?

Reply:
just makes our naming consistent since we can't omit the region for S3

Reply:
👍